Magento2/ruleset.xml

```diff
@@ -115,6 +115,13 @@
         <severity>10</severity>
         <type>error</type>
     </rule>
+    <rule ref="Magento2.Legacy.ObsoleteConfigNodes">
```

Comment:
I think the correct name is "ObsoleteNodeInConfig"

Reply:
@jcuerdo this is referring a to nodes specific to config, so it's also correct to call them this way. Also, old rule is called also config nodes.